lib/nodes/Base.ts

```diff
@@ -38,8 +40,8 @@ export class Base<S extends BlueshellState, E> {
 		}
 
 		try {
+			maybePublishTree(state, event, false);
```

Comment:
Can we have a tree publisher injected somehow so that the conditional only happens at boot (if no publish - inject Noop)

Reply:
Yep.. going with that..